PR to review Mobile menu branch

Hadi111jafari opened the pull request:
In this branch I have built the menu, the hamburger menu  and some functionality on it.

Hadi111jafari committed:
Create app.js file

Hadi111jafari committed:
Build mobile menu most parts

Hadi111jafari committed:
Add eventlistener to humburger and navmenu and fixed miss functionality

omsoo committed:
Update the nav link in app.js file and formatting

Work on updating the app.js file

Hadi111jafari committed:
fix linter errors

Hadi111jafari committed:
solved conflicts

Kingstalux commented on style.css:
```diff
@@ -12,41 +12,102 @@ body {
+    transition: 0.3s;
+    margin: 0;
+    padding: 28px;
+  }
```
- [ ] Good job so far 👍 It would be great if you could add the blur effect when the mobile menu is opened as shown in the Figma template 💯 
- [ ] Also it would be good to disable the scrolling of the page when the menu is open so that your design will follow the Figma template.
- Your design:
![Annotation 2022-03-08 172637](https://user-images.githubusercontent.com/85113567/157281592-754f3540-9d59-4ca5-a118-2a3b799e7b61.jpg)
- Figma design:
![Annotation 2022-03-08 172745](https://user-images.githubusercontent.com/85113567/157281628-338ebe75-4d68-4e23-9cf6-c84576d8b46b.jpg)

prolajumokeoni replied:
OPTIONAL

- [ ]  Kindly add the right background color, also I noticed the blurred background  is present but because there is no opacity on the background color I cant see this feature kindly make these changes, so your project looks like the original

Kingstalux reviewed (changes requested):
Hi @Hadi111jafari and @omsoo , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
![68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f6e59597451644c7177776c54653363634b682f67697068792e676966](https://user-images.githubusercontent.com/85113567/157282061-73a55e02-26f3-416f-9257-cf38c89ffc3f.gif)


## Highlights
✔️ Good commit messages
✔️ Mobile menu functions well

## Required Changes ♻️
_Check the comments under the review._
- [ ] You have to add the configurations of the [JavaScript linters](https://github.com/microverseinc/linters-config/tree/master/javascript) and make sure the checks show up on Github and they should be passing.
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Hadi111jafari committed:
Add the configurations of the JavaScript linters and fix the issues

Hadi111jafari committed:
applied required changes

Hadi111jafari committed:
stylelint error fixed

prolajumokeoni commented:
Hi @Hadi111jafari please don't delete comments from previous reviewers, if you don't understand the requested changes you can comment on the  thread for more clarification, I strongly advise you don't delete reviewers comments as this can hinder approval of your pull request, because as a second reviewer I have to ensure that you have effected all the requested changes.

Hadi111jafari commented:
Hi @prolajumokeoni ok by mistake I removed that was to understand well what to implement and that was about optional changes and I implemented optional also. And you can check if I added changes or not because it is all there, just  clarification message deleted because I thought no need to be here because I got to know what to implement.
Let me know if anything is there ?

prolajumokeoni reviewed (approved):
Hi @Hadi111jafari 


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


Congratulations! 🎉 

<br>

**Optional suggestions**

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

<br>
<br>


![approved-stamp](https://user-images.githubusercontent.com/69638013/157502000-998cc210-1280-4245-915f-d5bfb86f9634.gif)


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
